lib/eex/lib/eex.ex

```diff
@@ -39,12 +39,15 @@ defmodule EEx do
   All functions in this module accept EEx-related options.
   They are:
 
-    * `:line` - the line to be used as the template start. Defaults to 1.
     * `:file` - the file to be used in the template. Defaults to the given
       file the template is read from or to "nofile" when compiling from a string.
+    * `:line` - the line to be used as the template start. Defaults to 1.
+    * `:indentation` - (since v1.11.0) indentation to be added to each column.
```

Comment:
It seems off to define an option using its name. Also I may be wrong, but I while internally it probably gets applied to each column in each line, I think lines are more important than columns here. How about:
```suggestion
    * `:indentation` - (since v1.11.0) amount of spaces to be added before each line.
```

Reply:
yeah, I'm not happy super happy with the wording either but fwiw we define e.g. `:line` in terms of line :)

My first thought when I read your proposed wording is as if we somehow literally inject space characters before each line and thus adjust output but the option is about adjusting :column in the AST so that e.g. stacktraces would point to the exact spot where we e.g. call a function. (again, not saying that's what you meant but how I'd likely have interpreted it.)

Reply:
Right, someone could interpret it this way. Not sure how to improve it then, it could stay as is until someone else has a better idea.

Reply:
"an integer to be added to the column after every new line"?

Reply:
I went with:

```
    * `:indentation` - (since v1.11.0) an integer added to the column after every
       new line. Defaults to 0.
```

👍